README.md

```diff
@@ -166,15 +161,16 @@ colcon build --symlink-install --executor sequential --parallel-workers 1
 Now you can proceed to [run the exemplar](#run-the-exemplar).
 
 ## Run the exemplar
-TODO: What exactly happens here? Maybe write short sentences. What are arguments and what do I do with them? Which things do I have to run in separate terminals?
```

Comment:
Why are we not describing what the pieces of code do, especially howto use the arguments? I can add that if we want.

Reply:
We will, but I'm not sure how to handle it. Since the last README a lot has changed with the launch files and arguments and there's now a runner.  So we need to have it reflect all of that, and  then explain those commands.

Reply:
Okay, I'll write an initial draft, which can be updated when everything is finalized.